workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/request/RequestExecutor.java

```diff
@@ -29,11 +31,13 @@ public RequestExecutor(HttpClient httpClient) {
   @Override
   public void execute(ActivityExecutorContext<ExecuteRequest> execution) throws IOException {
     ExecuteRequest activity = execution.getActivity();
+    log.info("Executing request {} {}", activity.getMethod(), activity.getUrl());
 
     Response response =
         this.httpClient.execute(activity.getMethod(), activity.getUrl(), activity.getBody(),
             headersToString(activity.getHeaders()));
 
+    log.info("Received response {}", response.getContent());
```

Comment:
it might contain sensitive data (and be very big) so we might want to only log this in debug (or not log it at all)

Reply:
What about only logging the status code? the idea here is to see if the request has been fulfilled and if it succeeded or not, because we might not use the outputs in SWADL